autopush/db.py

```diff
@@ -266,6 +268,12 @@ def all_channels(self, uaid):
     def store_message(self, uaid, channel_id, data, headers, message_id, ttl):
         """Stores a message in the message table for the given uaid/channel with
         the message id"""
+        if data:
+            # Store Base64-encoded payloads as binary fields.
+            if not isinstance(data, bytes):
+                data = data.encode('utf-8')
+            data = Binary(urlsafe_b64decode(data))
```

Comment:
Why not leave it as b64 when its stored, and keep it a string field? Then we won't need to b64 encode it on the way out.